src/Tags/HeartbeatWrapper.lua

```diff
@@ -0,0 +1,5 @@
+local RunService = game:GetService("RunService")
+-- monkey patch for unit testing
+return {
+    Heartbeat = RunService.Heartbeat
+}
```

Comment:
After thinking about this again, I think we should instead get rid of this module and make it a property in the Fabric instance. `fabric.Heartbeat`